tests/Maker/FunctionalTest.php

```diff
@@ -282,12 +283,124 @@ public function getCommandTests()
             ])
         ];
 
-        yield 'auth_empty' => [MakerTestDetails::createTest(
-            $this->getMakerInstance(MakeAuthenticator::class),
-            [
-                // class name
-                'AppCustomAuthenticator',
-            ])
+        yield 'auth_empty_one_firewall' => [
+            MakerTestDetails::createTest(
+                $this->getMakerInstance(MakeAuthenticator::class),
+                [
+                    // class name
+                    'AppCustomAuthenticator',
+                ]
+            )
+                ->addExtraDependencies('security')
+                ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeAuthenticator')
+                ->setRequiredPhpVersion(70100)
```

Comment:
I don't think we need this. I added it for `MakeUser`, but that was because we were using MakeEntity in the background, which generates PHP 7.1 code (nullable types). But, in this case, the authenticator doesn't have nullable types, and updating the YAML obviously is ok.